lldb/test/Shell/Swift/caching.test

```diff
@@ -8,18 +8,15 @@
 # RUN:          -cache-compile-job -cas-path %t/cas -explicit-module-build \
 # RUN:          -module-name main -o %t/main
 
-# RUN: %lldb %t/main -s %t/lldb.script 2>&1 | FileCheck %s
+# RUN: sed "s|DIR|%/t|g" %t/lldb.script.template > %t/lldb.script
+# RUN: %lldb %t/main -s %t/lldb.script 2>&1 | FileCheck %s --check-prefix=CAS-LOAD --check-prefix=CHECK
 
-## Setup CAS and try loading from CAS
-# RUN: sed "s|DIR|%/t|g" %t/lldb.script.template > %t/lldb_2.script
-# RUN: %lldb %t/main -s %t/lldb_2.script 2>&1 | FileCheck %s --check-prefix=CAS-LOAD --check-prefix=CHECK
-
-## Check fallback to file system.
+## Check warning message
 # RUN: rm -rf %t/cas
-# RUN: %lldb %t/main -s %t/lldb_2.script 2>&1 | FileCheck %s --check-prefix=CAS-FALLBACK --check-prefix=CHECK
+# RUN: %lldb %t/main -s %t/lldb.script 2>&1 >/tmp/out.log
+#| FileCheck %s --check-prefix=CAS-FALLBACK --check-prefix=CHECK
```

Comment:
What does this do?

Note this crashes compiler the last time I checked due to SwiftShim is not loaded after deleting CAS. You need to take some of the `SwiftASTContext.cpp` change from https://github.com/swiftlang/llvm-project/pull/11983/changes (especially fixing the default_path) to make this work.